src/styles/_footer.scss

```diff
@@ -0,0 +1,73 @@
+$footer-background:#373d45;
```

Comment:
do we need a separate scss file for each component? there are some duplications of styles already between the header and footer.